<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>meetingRoomWidget</class>
 <widget class="QWidget" name="meetingRoomWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,20,1,5,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="meetingName_LB">
       <property name="text">
        <string>会议名称</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="host_LB">
       <property name="text">
        <string>主持人</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="20,6">
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="memberList_TW">
       <column>
        <property name="text">
         <string>成员</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>状态</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="camera_cb">
       <property name="text">
        <string>摄像头</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="mic_cb">
       <property name="text">
        <string>麦克风</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextBrowser" name="msgOutput_TB"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLineEdit" name="msgInput_LE"/>
     </item>
     <item>
      <widget class="QPushButton" name="sendMsg_PB">
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
